components/layout/blog-post/blog-post.hbs

```diff
@@ -0,0 +1,244 @@
+                        class="blog-article__blockquote-span">New York City</span></cite>
+            </blockquote><!-- .blog-article__post-blockquote -->
+
+            <h2>Show the Gap</h2>
+            <p>Start by determining where your team is falling short. When projects miss the mark, what are the most
+                common
+                skills needed to change those outcomes? Take a look at your goals. What additional expertise is required
+                to
+                achieve them? Once you have an overall view of the business needs, it’s time to take it to the
+                individual
+                level.</p>
+
+            <aside class="blog-article__post-aside">
+                <p class="p--body">Key Takeaway</p>
+                <p class="p--body">Be sure to explicitly define what success looks like–show the “gap” between the
+                    skills of today and
+                    the
+                    desired expertise. Drawing the path between their current role/knowledge and what they need to
+                    master to
+                    move to the next level not only gets staff engaged in their own development but prevents frustration
+                    on
+                    both sides if that career step does not materialize as quickly as they had in mind.</p>
+            </aside><!-- .blog-article__post-aside -->
+
+            <p>If you’re struggling to help someone on your team see that the promotion and compensation increase they
+                want does not match their current skills, ask more questions. A great question to ask staff was posed by
+                a leader in one of our roundtable discussions: “What are your skills today compared to [fill in the
+                blank] years ago?”</p>
+
+            <h2>Inspire Growth</h2>
+            <p>Tapping into career expectations and personal goals will uncover powerful ways to get staff involved in
+                their development. By asking questions, you will find expertise they may have thought wouldn’t fit their
+                role–but that you know would add value. Here’s an example:</p>
+            <p>Upon discovering that a designer on their team had a passion for illustration (which was not in his job
+                description), one InsideOut leader crafted opportunities to hone those skills which led to a companywide
+                repository of visual components that saved time and money–and ultimately led to that designer’s
+                promotion. Another leader pointed out that learning should be personal. When staff see what’s in it for
+                them to acquire new skills and how important those skills are to their success and that of the broader
+                organization, they embrace education and are thankful for your investment in them.</p>
+
+            <aside class="blog-article__post-aside">
+                <p class="p--body">Key Takeaway</p>
```

Comment:
Instead of manually baking this into an aside, at the moment, we have no variants to the pattern of asides in a blog post. What we could do here, like with the reading time deal, is use a ::before psuedo-element to apply the "Key Takeaway" label. If we have asides later that don't need that, then we can address those then.

Reply:
Done.